src/features/authPlaygroundAPI/authPlaygroundApiSlice.ts

```diff
@@ -0,0 +1,16 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+import { GetArkReauthQuery } from "../../lib/authentication/arkBaseQuery";
+
+const baseurl = "";
+
+export const authPlaygroundApi = createApi({
+  reducerPath: "authPlayground",
+  baseQuery: GetArkReauthQuery(baseurl),
```

Comment:
why 'get' semantic? is this a factory? If so use 'create'

RTKQ library has fetchBaseQuery, the module is arkBaseQuery, thus the question.